examples/evaluation/evaluate_with_model_validation.py

```diff
@@ -0,0 +1,73 @@
+import xgboost
```

Comment:
Nice! Can we add this to https://github.com/mlflow/mlflow/blob/b97155d7c4f210d63eae6588350068a7dc3f2db5/tests/examples/test_examples.py#L194?